src/main/java/sg/ncl/MainController.java

```diff
@@ -2640,44 +2640,24 @@ public String teamAppJoinFromTeamsPage(@PathVariable String teamName, Model mode
     //--------------------------Static pages for sign up--------------------------
 
     @RequestMapping("/team_application_submitted")
-    public String teamAppSubmit(Model model) {
-        model.addAttribute("loginForm", new LoginForm());
-        model.addAttribute("signUpMergedForm", new SignUpMergedForm());
+    public String teamAppSubmit() {
         return "team_application_submitted";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)